pom.xml

```diff
@@ -71,11 +71,11 @@
 
     <properties>
         <!-- Dependency Versions -->
-        <spring-cloud-commons.version>2.1.2.RELEASE</spring-cloud-commons.version>
-        <spring-cloud-netflix.version>2.1.2.RELEASE</spring-cloud-netflix.version>
-        <spring-cloud-openfeign.version>2.1.2.RELEASE</spring-cloud-openfeign.version>
-        <spring-cloud-bus.version>2.1.2.RELEASE</spring-cloud-bus.version>
-        <spring-cloud-gateway.version>2.1.2.RELEASE</spring-cloud-gateway.version>
+        <spring-cloud-commons.version>2.1.3.RELEASE</spring-cloud-commons.version>
+        <spring-cloud-netflix.version>2.1.3.RELEASE</spring-cloud-netflix.version>
+        <spring-cloud-openfeign.version>2.1.3.RELEASE</spring-cloud-openfeign.version>
+        <spring-cloud-bus.version>2.1.3.RELEASE</spring-cloud-bus.version>
+        <spring-cloud-gateway.version>2.1.3.RELEASE</spring-cloud-gateway.version>
```

Comment:
Is it possible to add an attribute here like this?
`<spring-cloud-component.version>2.1.3.RELEASE</spring-cloud-component.version>`

Other spring cloud components introduce versions through this property, like this
`<spring-cloud-commons.version>${spring-cloud-component.version}</spring-cloud-commons.version>`